src/main/java/hudson/plugins/gradle/injection/GradleEnterpriseExceptionTaskListenerDecoratorFactory.java

```diff
@@ -1,34 +1,43 @@
 package hudson.plugins.gradle.injection;
 
 import hudson.Extension;
+import hudson.model.Queue;
 import hudson.model.Run;
+import org.jenkinsci.plugins.workflow.flow.FlowExecutionOwner;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
 import org.jenkinsci.plugins.workflow.log.TaskListenerDecorator;
-import org.jenkinsci.plugins.workflow.steps.DynamicContext;
 
-import javax.annotation.CheckForNull;
 import javax.annotation.Nonnull;
 import java.io.IOException;
 import java.io.OutputStream;
+import java.io.Serializable;
+import java.util.logging.Level;
+import java.util.logging.Logger;
 
 @SuppressWarnings("unused")
 @Extension
-public class GradleEnterpriseExceptionTaskListenerDecoratorFactory extends DynamicContext.Typed<TaskListenerDecorator> {
+public class GradleEnterpriseExceptionTaskListenerDecoratorFactory implements TaskListenerDecorator.Factory {
 
-    @Nonnull
-    @Override
-    protected Class<TaskListenerDecorator> type() {
-        return TaskListenerDecorator.class;
-    }
+    private static final Logger LOGGER = Logger.getLogger(GradleEnterpriseExceptionTaskListenerDecoratorFactory.class.getName());
 
-    @CheckForNull
```

Comment:
❓ Why it was removed? The method returns `null`.

Reply:
Just got lost during refactoring, thanks